### DISCUSSION TITLE
How to detect hover events (enter, exit)?

### DISCUSSION THREAD

**Discussion by projectitis** [flame-engine/flame#2658] [Q&A] [answered]
Hi all,

I've been going through the code and the docs, and I'm quite confused about how to detect hover events, or whether this has been removed. I was about to make a feature suggestion to add MouseMovementDetector `onMouseOver` and `onMouseout`, but I suspect there is already a way to do this!

There are variously different references to deprecated mixins, including `Hoverable`. 'HasHoverables` and `HoverableCallbacks`. 

Also, there are some promising data structures (`PointerEnterEvent` and `PointerExitEvent`) but no docs on how/where these are surfaced or detected :) 

Any help on how to detect hover enter/exit appreciated!

**Comment by spydon** [comment, marked as answer]
Hi, in the new event system that doesn't exist yet, it is underway in #1834 and someone else wanted to help out with building that so hopefully we'll have that in place soon.
